tests/fixtures/issues/issue-900.php

```diff
@@ -15848,112 +15848,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-900-LC' => [
-        'ua' => 'Mozilla/6.0 (Macintosh; I; Intel Mac OS X 11_7_9; de-LI; rv:1.9b4) Gecko/2012010317 Firefox/10.0a4',
```

Comment:
I believe this one ended up as an "invalid-version" since there are a number of issues with it (bad OSX version (currently), `rv` and `Firefox` versions don't match, `Gecko` tail isn't correct).